packages/codebytes/src/libs/eventTracking.ts

```diff
@@ -0,0 +1,18 @@
+import { createTracker } from '@codecademy/tracking';
+
+const IS_DEV = process.env.NODE_ENV === 'development';
+
+// TODO: confirm tracking details and implementation DISC-447
+const tracker = createTracker({
+  apiBaseUrl:
+    typeof window === undefined
+      ? 'https://www.codecademy.com'
```

Comment:
window will be undefined in an inframe so this works for forums?

Reply:
for iframes, window will not be `undefined` because its referring to the iframe window and from that we can grab `window.location.origin`